debugging-via-dotnet-dump/test.sh

```diff
@@ -554,7 +556,17 @@ cat dump.out
 id=$(tail -1 dump.out | cut -d' ' -f 2)
 dump-analyze "objsize ${id}" > dump.out
 cat dump.out
-grep -E '^sizeof\([0-9a-fA-F]+\) += +[0-9]+ \(0x[0-9a-fA-F]+\) bytes' dump.out
+if grep -F 'transitively keep alive' dump.out; then
```

Comment:
Is it just a version change that causes this difference, or something about the running env that gets dumped? I ask because checking for aversion number or something might better express why this is here.

Reply:
The `dotnet dump` tool is undergoing significant development (essentially, a rewrite from C++ to C#), resulting in a number of recent changes. The output change is a result of that: older versions printed the object size, the newer versions also show method tables and other information. The changes do bump the version number of the tool ([`7.0.410101`](https://www.nuget.org/packages/dotnet-dump) -> `7.0.418102`).

Checking for the this patch-version boundry seemed too brittle and so I decided to check the output instead.